services/frontend/src/components/profileForms/formTitle/FormTitleView.jsx

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { FormattedMessage } from "react-intl";
+import { Typography } from "antd";
+
+import "./FormTitle.scss";
+
+const { Title, Text } = Typography;
+
+const FormTitleView = ({
+  title,
+  formType,
+  stepNumber,
+  fieldsChanged,
+  extra,
+}) => {
+  if (formType === "create") {
+    return (
+      <Title level={2} className="pgf-formTitle">
+        {stepNumber}. {title}
+      </Title>
+    );
+  }
+  return (
+    <>
+      <Title level={2} className="pgf-formTitle">
+        {title}
+        {fieldsChanged && (
+          <Text className="pgf-unsavedText">
```

Comment:
the class name pgf is defined where? I think it should be defined in the associated scss file for this component, and probably get renamed